pkgs/applications/graphics/inkscape/default.nix

```diff
@@ -38,6 +38,7 @@
 , python3
 , substituteAll
 , wrapGAppsHook
+, libepoxy
 , zlib
 }:
 let
```

Comment:
Looking at the [list of required Python modules](https://inkscape.gitlab.io/extensions/python-dependencies.txt) linked from changelog, we also need the following:

- pyparsing (used by newly added hpgl parser extension)

The following modules listed do not appear to be used by anything:

- tinycss2
- filelock

Reply:
I tried running `docs/poetry.sh` in `share/extensions` directory but could not reproduce the list – I got the following on both `INKSCAPE_1_3` and `master` bracnches:

```markdown
# Packages Used Directly

You must always include these in your package:

 * cssselect 1.2.0
 * lxml 4.9.3
 * numpy 1.25.2
 * packaging 23.1
 * pillow 9.5.0
 * pygobject 3.44.1
 * pyparsing 3.1.1
 * pyserial 3.5
 * scour 0.37


# Packages Used Indirectly

You may or may not need to include these, depending on if your packaging system automatically pulls in sub-deps:

 * pycairo >=1.16,<2.0
 * six >=1.9.0
```